minicluster/src/test/java/org/apache/accumulo/minicluster/MiniAccumuloClusterTest.java

```diff
@@ -78,8 +78,10 @@ public static void setupMiniCluster() throws Exception {
 
     MiniAccumuloConfig config = new MiniAccumuloConfig(testDir, ROOT_PASSWORD).setJDWPEnabled(true);
     config.setZooKeeperPort(0);
+    @SuppressWarnings("deprecation")
+    String workQThreadsProp = Property.TSERV_WORKQ_THREADS.getKey();
     HashMap<String,String> site = new HashMap<>();
-    site.put(Property.TSERV_WORKQ_THREADS.getKey(), "2");
+    site.put(workQThreadsProp, "2");
```

Comment:
This test probably does not care which prop is used ,so could switch to a non deprecated prop.

```suggestion
    site.put(Property.TSERV_COMPACTION_WARN_TIME.getKey(), "5m");
```